src/overlays/actors/ovl_Obj_Mure/z_obj_mure.c

```diff
@@ -26,43 +40,375 @@ const ActorInit Obj_Mure_InitVars = {
     (ActorFunc)ObjMure_Update,
     NULL,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Mure/func_80B98AA0.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Mure/func_80B98B1C.s")
+typedef enum {
+    /* 0 */ OBJMURE_TYPE_GRASS,
+    /* 1 */ OBJMURE_TYPE_UNDEFINED,
+    /* 2 */ OBJMURE_TYPE_FISH,
+    /* 3 */ OBJMURE_TYPE_BUGS,
+    /* 4 */ OBJMURE_TYPE_BUTTERFLY
+} ObjMureType;
+
+f32 sZClip[] = { 1600.0f, 1600.0f, 1000.0f, 1000.0f, 1000.0f };
+
+s32 sMaxChildSpawns[] = { 12, 9, 8, 0 };
+
+s16 sSpawnActorIds[] = { ACTOR_EN_KUSA, 0, ACTOR_EN_FISH, ACTOR_EN_INSECT, ACTOR_EN_BUTTE };
+
+s16 sSpawnParams[] = { 0, 2, -1, 0, -1 };
+
+// sInitChain
+InitChainEntry D_80B99A8C[] = {
```

Comment:
```suggestion
static InitChainEntry sInitChain[] = {
```
Probably make all data static btw